.travis.yml

```diff
@@ -4,6 +4,8 @@ before_install:
 - gem install slather -N 
 
 script:
-- xctool -project libPhoneNumber.xcodeproj -scheme libPhoneNumberiOSTests -sdk iphonesimulator clean build test ONLY_ACTIVE_ARCH=NO GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES  
+- xcodebuild -project libPhoneNumber.xcodeproj -scheme libPhoneNumberiOSTests clean
```

Comment:
Can we break this out into a separate PR (given that it has nothing to do with the short number and emergency number)

Reply:
Just created a pull request to separate build changes. https://github.com/iziz/libPhoneNumber-iOS/pull/222